client/components/RunningGraph.vue

```diff
@@ -33,61 +37,72 @@
  import { eventBus } from '../client';
  import { getBackgroundColorFor, getBorderColorFor, lineWidthInPixels, pointRadiusInPixels} from '../client'; 
  import { truncate} from '../pages/utilities.js'; 
+ import { isPathIn} from '../pages/utilities.js'; 
```

Comment:
Can be condensed see https://github.com/fnproject/ui/pull/37/files#diff-a3db016c87a936e401dcbc268b43db7cR40

Reply:
Done